pkg/api/errors/errors.go

```diff
@@ -136,30 +138,32 @@ func NewUnauthorized(reason string) error {
 }
 
 // NewForbidden returns an error indicating the requested action was forbidden
-func NewForbidden(kind, name string, err error) error {
+func NewForbidden(qualifiedResource unversioned.GroupResource, name string, err error) error {
```

Comment:
@smarterclayton The resource is forbidden, no kind determination was made because that requires the body.